support-frontend/assets/components/subscriptionCheckouts/personalDetails.jsx

```diff
@@ -0,0 +1,92 @@
+// @flow
+import React from 'react';
+import { firstError } from '../../helpers/subscriptionsForms/validation';
```

Comment:
I'm not sure I like this import style, could we stick to absolute imports when they start at the root?

Reply:
Yup Intellij added this automatically but I agree that absolute imports are nicer